lib/aptible/cli/subcommands/restart.rb

```diff
@@ -8,10 +8,34 @@ def self.included(thor)
             include Helpers::App
 
             desc 'restart', 'Restart all services associated with an app'
+            option :'simulate-oom',
```

Comment:
I think this can/should be just `option :simulate_oom` for consistency (Thor will expect the argument with a hyphen on the command line). Otherwise, looks great and this functionality is excellent.